cx_Freeze/macdist.py

```diff
@@ -177,6 +184,15 @@ def initialize_options(self):
 
     def finalize_options(self):
         self.include_frameworks = normalize_to_list(self.include_frameworks)
+        if self.plist_items is None: self.plist_items = []
```

Comment:
With change in initialize_option this should be changed too, like in https://github.com/marcelotduarte/cx_Freeze/blob/master/cx_Freeze/dist.py#L221

Reply:
Made the change.  

One thing I'll note: if we change the list option handling to be like in build_exe, then IDEs won't be able to understand that the option members exist on the objection.  For that reason, I would slightly prefer just listing out all the options explicitly in initialize_options()--but I do not care strongly.

Reply:
For IDE you can use typing hints, as PEP 526, without an initial value.

Reply:
Would you be okay listing all the List-options in initialize_options(), to hint their types?

Reply:
I meant, without changing what you've already done,
```
class bdist_mac(Command):
    plist_items: List[...]
```

Reply:
Makes sense.  Done.